Create ui-c8y-1021-0-0-update-branding-7255.md (#2577)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-0-0-update-branding-7255.md

---
date: ""

> title: update branding (#7255)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Update branding colors and logo
```

> title: Update branding colors and logo

Reviewer: ```suggestion
title: Updated branding colors and logo
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61407
version: 1021.0.0
---

> update branding (#7255)

Author: The Claude AI suggested to use the following body:
```suggestion
As part of ongoing efforts to modernize the look and feel of the application, the branding colors and logo have been updated. The previous color scheme and logo were outdated and did not align with current design trends. With this change, users will notice a refreshed visual appearance throughout the application, featuring a new color palette and an updated logo. This update brings a more contemporary and professional look to the application without impacting existing functionality or user workflows.
```

> As part of ongoing efforts to modernize the look and feel of the application, the branding colors and logo have been updated. The previous color scheme and logo were outdated and did not align with current design trends. With this change, users will notice a refreshed visual appearance throughout the application, featuring a new color palette and an updated logo. This update brings a more contemporary and professional look to the application without impacting existing functionality or user workflows.

Reviewer: ```suggestion
As part of ongoing efforts to modernize the look and feel of the {{< product-c8y-iot >}} UI, the branding colors and the logo have been updated. The previous color scheme and logo were outdated and did not align with current design trends. This change provides a refreshed visual appearance throughout the {{< product-c8y-iot >}} UI, without impacting existing functionality or user workflows.
```

> As part of ongoing efforts to modernize the look and feel of the application, the branding colors and logo have been updated. The previous color scheme and logo were outdated and did not align with current design trends. With this change, users will notice a refreshed visual appearance throughout the application, featuring a new color palette and an updated logo. This update brings a more contemporary and professional look to the application without impacting existing functionality or user workflows.

Reviewer: ```suggestion
As part of ongoing efforts to modernize the look and feel of the {{< product-c8y-iot >}} UI, the branding colors and the logo have been updated. The previous color scheme and logo were outdated and did not align with current design trends. This change provides a refreshed visual appearance throughout the {{< product-c8y-iot >}} UI, without impacting existing functionality or user workflows.
```
